docs/content/users/quickstart.md

```diff
@@ -470,7 +510,15 @@ While the generic `php` project type is [ready to go](./project.md) with any CMS
 
 ## Configuration Files
 
-The [`ddev config`](../users/usage/commands.md#config) command attempts to create a CMS-specific settings file pre-populated with DDEV credentials.
+The [`ddev config`](../users/usage/commands.md#config) and `ddev start` commands attempt to create a CMS-specific settings file pre-populated with DDEV credentials. If you don't want DDEV to do this, use `settings_management_disabled: true` in the `.ddev/config.yaml`. `
```

Comment:
Fix the name of the setting and lean on reference material to help people get comfortable with the concept of config settings and the structure of the docs:

```suggestion
The [`ddev config`](../users/usage/commands.md#config) and `ddev start` commands attempt to create a CMS-specific settings file pre-populated with DDEV credentials. If you don't want DDEV to do this, set the [`disable_settings_management`](../users/configuration/config.md#disable_settings_management) config option to `true`.
```

Reply:
Wow, glad you caught that!